Document file_types filtering (#8506)
discord/discord-api-docs: developers/components/reference.mdx

|-----------------|------------------|-----------------------------------------------------------------------------------------------------------------|
| type            | integer          | `19` for file upload                                                                                            |
| id?             | integer          | Optional identifier for component                                                                               |
| custom_id       | string           | ID for the file upload; 1-100 characters                                                                        |
| min_values?\*   | integer          | Minimum number of items that must be uploaded (defaults to 1); min 0 (see note), max 10                         |
| max_values?     | integer          | Maximum number of items that can be uploaded (defaults to 1); max 10                                            |
| required?       | boolean          | Whether the file upload requires files to be uploaded before submitting the modal (defaults to `true`)          |

> | file_types?\*\* | array of strings | File types to filter for; can be `image`, `video`, `audio` or any dot-prefixed extension such as `.pdf`; max 10 |

Reviewer: Missing an oxford comma for a list of three (or more) items.
```suggestion
| file_types?\*\* | array of strings | File types to filter for; can be `image`, `video`, `audio`, or any dot-prefixed extension such as `.pdf`; max 10 |
```

\* `min_values` must be either omitted or at least `1` if `required` is omitted or `true`.

> \*\* `file_types` will only check the extension on the filename. Check [File Groups](/developers/reference#file-groups) for more details.

Reviewer: Sounds a bit colloquial to me, but not a strict problem.
```suggestion
\*\* `file_types` only matches against the file extension. See [File Groups](/developers/reference#file-groups) for details.
```
